webdriver/tests/element_click/link.py

```diff
@@ -0,0 +1,97 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
+
+
+# 14.1 Element Click Link Element Tests
+
+def test_numbers_link(session):
+    url = "/webdriver/tests/element_click/support/input.html"
+    session.url = inline("<a href=%s>123456</a>" % url)
```

Comment:
Feel free to ignore but it would be good to keep the usage of `format` instead of mixing with `%`.

Reply:
fixed